packages/admin-ui/ui/src/components/molecules/select/next-select/components/single-value.tsx

```diff
@@ -13,28 +16,50 @@ const SingleValue = <
   className,
   isDisabled,
   data,
+  getValue,
 }: SingleValueProps<Option, IsMulti, Group>) => {
   const prefix = hasPrefix(data) ? data.prefix : null
 
+  const isEllipsisActive = (e: HTMLDivElement | null) => {
+    if (!e || !(e.offsetParent as HTMLDivElement)?.offsetWidth) {
+      return false
+    }
+
+    return (e.offsetParent as HTMLDivElement).offsetWidth < e.scrollWidth
+  }
+
+  const toolTip = !getValue().length
+    ? null
+    : ((getValue()?.[0] as { label: string })?.label as string) ?? null
```

Comment:
comment: This looks a bit funky. If possible, we should make this a tiny bit more readable 😄

Reply:
created a short util method instead to extract this in the component for readability